jaqpotpy/descriptors/tests/test_mordred_descriptors.py

```diff
@@ -14,55 +14,36 @@ def setUp(self):
         Set up tests.
         """
         from rdkit import Chem
```

Comment:
If you have time, move all imports at the beginning of the script. I don't get why imports were placed inside functions